migration/v0.26.x-v0.27.x/upgrade-apps.md

```diff
@@ -0,0 +1,5 @@
+# Upgrade v0.26.x to v0.27.x
+
+## Steps
+
+1. Update velero CRD's.
```

Comment:
And the command to do it :)

Reply:
I think I made a confusion here
I have already replaced the entire velero directory with the contents from the new release, this includes CRD's
The only thing that I did to apply it then it was to sun `bin/ck8s apply {sc|wc}` and I did not add this command as I thought it will part of the migration steps for the new release/patch that includes this ...
Should I now remove this step `Update velero CRD's.` ?

Reply:
Helm will not update CRDs. So you have to run `kubectl apply` on that crds directory to update them.

Reply:
OK, after inspecting it the Velero helm chart includes a job that updates the CRDs, so yes you can remove the step.

Reply:
If this is taken care of by the apply step, I think you can remove it completely.

Reply:
I have removed it